video_xblock/static/html/studio-edit-tab.html

```diff
@@ -53,6 +53,11 @@
     </li>
 {% endfor %}
 
+<li class="field comp-setting-entry metadata_entry">
+    <label class="label setting-label">Component Location ID</label>
```

Comment:
Why did you apply the `label` tag? As I remember correctly, <label> tag establishes a relationship between a specific label, which is usually text, and a form element (<input>, <select>, <textarea>). As I see you do not have these relations. I guess `span` tag is a good candidate to use here.